docs/deploy-with-olm.md

```diff
@@ -2,6 +2,8 @@
 
 The local-storage Operator is avaialble in OLM, this makes it easy to consume local storage on your OCP cluster.
 
+**Warning: These docs are for internal development and testing. Use https://docs.openshift.com/container-platform/4.2/storage/persistent-storage/persistent-storage-local.html docs for installation on OCP**
```

Comment:
I wonder how stable is `latest` in this: https://docs.openshift.com/container-platform/latest/storage/persistent-storage/persistent-storage-local.html
@huffmanca, are there any guarantees about link names?

Reply:
`latest` should be acceptable here - it will redirect to the latest published version. If it directs to a password-protected unpublished version, then that's a bug.

Reply:
fixed